tools/settings.py

```diff
@@ -52,6 +52,7 @@
     'SDL2_MIXER_FORMATS',
     'SDL2_IMAGE_FORMATS',
     'USE_SQLITE3',
+    'USE_CONTRIB_PORT'
```

Comment:
Since settings are not (currently) additive, should this new setting be `-sUSE_CONTRIB_PORTS=` or `-sUSE_PORTS=`?

Reply:
Yes I agree. I will use `-sUSE_PORTS`.

Reply:
Or maybe just `-sPORTS=`?

Reply:
Love it!